site/source/user-manual/upgrade-02/upgrade-kit.rst

```diff
@@ -16,14 +16,9 @@ This guide will cover how to upgrade from EmbassyOS version 0.2.x to version 0.3
 Backing up
 ----------
 
-.. caution:: It is prudent to back up your Embassy before migrating in case something doesn't work correctly, or there is an accident
+We recommend creating fresh backups of your 0.2.x services before getting started with your upgrade kit.
 
-If you're unsure how to do this - please follow `this <https://youtube.com/watch?v=_QJXgnE90ko>`_ guide.
-
-.. youtube:: _QJXgnE90ko
-   :width: 100%
-
-With your Embassy safely backed up, it's time to proceed with the migration.
+*Please don't use the microSD card we sent you as a backup device!*
```

Comment:
This is a good place for a "warning" "admonition" https://sphinx-theme.scylladb.com/stable/examples/admonitions.html